### PR TITLE
typo fixed in 10-std.tex

### PR DESCRIPTION
**Pull request by iamrajiv**

### CODE REVIEW

File: HaxeManual/10-std.tex [hunk 1]
```diff
@@ -196,7 +196,7 @@ \section{Regular Expressions}
 var r = new EReg("haxe", "i");
 \end{lstlisting}
 
-First argument is a string with regular expression pattern, second one is a string with \emph{flags} (see below).
+The first argument is a string with regular expression pattern, the second one is a string with \emph{flags} (see below).
```
Comment (Aurel300) [inline HaxeManual/10-std.tex]: `... with a regular expression pattern, ...`

File: HaxeManual/10-std.tex [hunk 2]
```diff
@@ -278,7 +278,7 @@ \subsection{Split}
 \subsection{Map}
 \label{std-regex-map}
 
-The \expr{map} method of a regular expression object can be used to replace matched substrings using a custom function. This function takes a regular expression object as its first argument so we may use it to get additional information about the match being done and do conditional replacement. For example:
+The \expr{map} method of a regular expression object can be used to replace matched substrings using a custom function. This function takes a regular expression object as its first argument so we may use it to get additional information about the match being done and do the conditional replacement. For example:
```
Comment (Aurel300) [inline HaxeManual/10-std.tex]: No.

File: HaxeManual/10-std.tex [hunk 3]
```diff
@@ -336,7 +336,7 @@ \subsection{Integer Math}
 	var intDivision = Std.int(6.2/4.7);
 \end{lstlisting}
 
-\todo{I think C++ can use integer operatins, but I don't know about any other targets. Only saw this mentioned in an old discussion thread, still true?}
+\todo{I think C++ can use integer operations, but I don't know about any other targets. Only saw this mentioned in an old discussion thread, still true?}
```
Comment (Aurel300) [inline HaxeManual/10-std.tex]: Please don't worry about fixing `\todo`, they don't show up on the webpage, and many are rather outdated anyway.

File: HaxeManual/10-std.tex [hunk 4]
```diff
@@ -348,15 +348,15 @@ \subsection{Extensions}
 \section{Lambda}
 \label{std-Lambda}
 
-\define{Lambda}{define-lambda}{Lambda is a functional language concept within Haxe that allows you to apply a function to a list or \tref{iterators}{lf-iterators}. The Lambda class is a collection of functional methods in order to use functional-style programming with Haxe.}
+\define{Lambda}{define-lambda}{Lambda is a functional language concept within Haxe that allows you to apply a function to a list or \tref{iterators}{lf-iterators}. The Lambda-class is a collection of functional methods in order to use functional-style programming with Haxe.}
```
Comment (Aurel300) [inline HaxeManual/10-std.tex]: No, why? The name of the class is `Lambda`.

File: HaxeManual/10-std.tex [hunk 5]
```diff
@@ -400,7 +400,7 @@ \section{Lambda}
 }
 \end{lstlisting} 
 
-This example demonstrates the Lambda count, has, foreach and fold function on a set of ints.
+This example demonstrates the Lambda count, has, for each and fold function on a set of ints.
```
Comment (Aurel300) [inline HaxeManual/10-std.tex]: No, `foreach` is the name of the function. If anything, they should be in `\expr`.

File: HaxeManual/10-std.tex [hunk 6]
```diff
@@ -522,7 +522,7 @@ \section{Template}
 Place the template-content in a new file called \ic{sample.mtt}, add \ic{--resource sample.mtt@my_sample} to the compiler arguments and retrieve the content using \expr{haxe.Resource.getString}.
 \haxe{assets/TemplateResource.hx}
 
-When running the template system on the server side, you can simply use \expr{neko.Lib.print} or \expr{php.Lib.print} instead of trace to display the HTML template to the user.
+When running the template system on the server side, you can simply use \expr{neko.Lib.print} or \expr{php.Lib.print} instead of a trace to display the HTML template to the user.
```
Comment (Aurel300) [inline HaxeManual/10-std.tex]: No article, `\expr{trace}`.

File: HaxeManual/10-std.tex [hunk 7]
```diff
@@ -890,7 +890,7 @@ \subsection{Threading}
 
 \since{4.0.0}
 
-A \href{https://api.haxe.org/sys/thread/Thread.html}{unified threading API} is available on some sys targets. The compile-time define \expr{target.threaded} is set when the API is available. The API allows very simple creation of threads from functions:
+A \href{https://api.haxe.org/sys/thread/Thread.html}{unified threading API} is available on some sys targets. The compile-time define \expr{target.threaded} is set when the API is available. The API allows a very simple creation of threads from functions:
```
Comment (Aurel300) [inline HaxeManual/10-std.tex]: No, it allows creation.

**Comment by iamrajiv** [comment]
thankyou @Aurel300 for suggestions

**Pull request merged**